lib/macro/macro-action-variable.cpp

```diff
@@ -68,6 +68,7 @@ static void modifyNumValue(Variable &var, double val, const bool increment)
 	if (!current.has_value()) {
 		return;
 	}
+
```

Comment:
Sorry for being pedantic, but is that change really needed? :)

Reply:
This one in particular or adding some spacing for readability in general?

Reply:
It's in the increment function, which is directly related. There were even changes before, but this was the only one I left in the end.

Reply:
>It's in the increment function, which is directly related.

This is not what I meant.
Adding an empty line to `macro-action-variable.cpp` is not related to the change `Check for lower/upper GetDouble/GetInt limits`.

>There were even changes before, but this was the only one I left in the end. Do you want for me to get rid of this single line addition?

Feel free to put those types of changes in a separate commit or even a separate PR, if you like. :)
To me this change simply does not fit into `Check for lower/upper GetDouble/GetInt limits` (even with `+ small refactors` added at the end).

I am probably just being overly cautious, but I just want to be sure, that in case any issues pop up, it is as easy as possible to track down what was changed and why.
Having those style changes in `macro-action-variable.cpp` mixed in with `Check for lower/upper GetDouble/GetInt limits` might hinder that.

I hope you understand! :)

And I am not trying to be dismissive or unappreciative of you putting in time to improve the plugin!
I value that a lot - so thank you!

Reply:
I understand ofc, but whitespace change does nothing in terms of debugging IMO. I'll move it to another commit.